src/index.html

```diff
@@ -17,6 +21,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-container">
+        <div class="logo-container">
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="MOYO Logo"
+              class="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav__container">
+          <ul>
+            <li>
+              <a
+                class="nav__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+                data-qa="hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
```

Comment:
Remember about correct indentation between parent and child elements. Each 'li' and 'a' tag should be indented with 2 spaces relative to its parent 'ul' tag, and 'a' should be indented relative to its parent 'li' tag.